SemesterReview.tex

```diff
@@ -0,0 +1,38 @@
+\documentclass{scrartcl}
+
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+\usepackage{amsmath}
+
+\title{Semester Review Report}
+
+
+
+
+\author{DH185421}
+
+\begin{document}
+
+\maketitle
+
+\abstract{This report will be looking at the skills that I have personally been lacking and what next steps I should take towards improving them.}
+
+
+
+
+
+\section{Key Skill 1 - Communication}
+
+Communication is a key part for a student to use to be able to get and give information regarding the course and work set. For me the most significant aspect was to be able to know what I am doing and know where I am going wrong. Alternatively this is perhaps due to some sort of anxiety I get when asking for help as I don’t want to burden anyone with my problems. Equally this is probably shown by the lack of quality and quantity of my work in the first semester. Having experienced not knowing what to do until it's nearly too late I now feel like I need to ask for help when I need it, additionally I have learned that I need to develop a rapport with my lecturers and classmates so I am not afraid to ask. I have slightly developed my skills in communication by helping and talking about the work with a classmate. This skill will be essential to me as a learner and a practitioner because without it I will not fully understand what is expected of me/work. As a next step I will need to communicate more with my lecturer to make sure I am on task.
```

Comment:
Be more specific with your proposed action(s) and how it is SMART. For example, as an action you could set up a regular meeting time to check in with tutors and measure your success based on your attendence.